src/components/Button/Button.tsx

```diff
@@ -57,8 +58,21 @@ const Button = styled.button<ButtonProps>`
 
   &:active:hover,
   &:hover {
-    background: ${props => props.theme.button.background.hover[`${props.color}`]};
+    background: ${props => props.theme.button.background.hover.main[props.color || 'primary']};
   }
+
+  ${props =>
+    props.variant === 'outline' &&
+    css`
+      background: ${props.theme.button.background.main.outline};
+      border: solid 2px ${props.theme.button.background.main[props.color || 'primary']};
+      color: ${props.theme.button.background.main[props.color || 'primary']};
+
+      &:active:hover,
```

Comment:
maybe we need to put :focus also on the same hover effects ? ( for better accessibility )